TaskService/TaskService.cs

```diff
@@ -38,9 +38,14 @@ public sealed partial class TaskService : Component, ISupportInitialize, System.
 	{
 		internal static readonly bool LibraryIsV2 = Environment.OSVersion.Version.Major >= 6;
 		private static readonly Version v1Ver = new Version(1, 1);
-		private static Guid CLSID_Ctask = Marshal.GenerateGuidForType(typeof(V1Interop.CTask));
+#if NETSTANDARD
+        private static Guid CLSID_Ctask = typeof(V1Interop.CTask).GUID;

```

Comment:
For these, just replace mine with your new typeof(x).GUID style. It works for both.